scroll-animations-1/Overview.bs

```diff
@@ -789,28 +770,66 @@ follows:
 
 </div>
 
-### The effective scroll range of a {{ScrollTimeline}} ### {#effective-scroll-range-algorithm}
-
-The procedure to calculate <dfn>effective scroll range</dfn> of a
-{{ScrollTimeline}} is as follows:
-
-1.  Run the procedure to [=resolve a scroll timeline offset=] for both {{start}}
-    and {{end}}.
-
-1.  Calculate [=effective scroll range=] as follow:
-    <div class="switch">
-    :   If [=effective start offset=] or [=effective end offset=] is null.
-    ::  The [=effective scroll range=] is null.
-
-    :   Otherwise
-    ::  The [=effective scroll range=] is the result of evaluating the following
-        expression:
-        <blockquote>
-        `[=effective end offset=] - [=effective start offset=]`
-        </blockquote>
-
-    </div>
-
+### The effective scroll offsets of a {{ScrollTimeline}} ### {#effective-scroll-offsets-algorithm}
+
+The <dfn>effective scroll offsets</dfn> are calculated by the procedure to
+<dfn>resolve scroll timeline offsets</dfn> from {{ScrollTimeline/scrollOffsets}}
+as follows:
+
+1.  Let <var>effective scroll offsets</var> be an empty list of
+    <a>effective scroll offset</a>s.
+1.  Let <var>first offset</var> be true.
+1.  If {{ScrollTimeline/scrollOffsets}} is empty
+    1.  Run the procedure to [=resolve a scroll timeline offset=] for 'auto'
+        with the <var>is first</var> flag set to <var>first offset</var> and
+        add the resulted value into <var>effective scroll offsets</var>.
+    1.  Set <var>first offset</var> to false.
+    1.  Run the procedure to [=resolve a scroll timeline offset=] for 'auto'
+        with the <var>is first</var> flag set to <var>first offset</var> and
+        add the resulted value into <var>effective scroll offsets</var>.
+1.  If {{ScrollTimeline/scrollOffsets}} has one element
```

Comment:
Should this say:

If {{ScrollTimeline/scrollOffsets}} has at least one element?

Reply:
This describes how to handle single scroll offset - add 'auto' for as a first element, the provided offset will be added as part of the next statement.

Reply:
Right. I did understand that at some point, then I confused myself.

Then probably this should say:

If {{ScrollTimeline/scrollOffsets}} has _exactly_ one element

Just to make it clear that we're not intending the "at least" meaning. (Which would prepend 'auto' for any non-empty list I guess).

Reply:
Done